kore/src/Kore/Builtin/Map/Map.hs

```diff
@@ -266,4 +269,4 @@ asTermLike builtin =
         :: (TermLike variable, MapValue (TermLike variable))
         -> TermLike variable
     element (key, MapValue value) =
-        mkApplySymbol elementSymbol [key, value]
+        mkApplySymbol (fromElement elementSymbol) [key, value]
```

Comment:
Partial functions should be avoided. Can this be refactored?

Reply:
see discussion above